### ISSUE TITLE
UNMS dont start after reboot or uninstall & reinstall

### ISSUE THREAD

**Issue by fgrosswig** [Ubiquiti-App/UNMS#118]
What is actually the issue with this nginx ???

I can not log in to the UNMS. It worked 1 time after fresh install and tried to see how the server beheaves after reboot i am not able to connect anymore.

See log of nginx:

{"log":"2018/02/02 12:29:32 [error] 25#25: *1 connect() failed (111: Connection refused) while connecting to upstream, client: 172.19.0.1, server: , request: \"GET / HTTP/1.1\", upstream: \"http://172.19.0.3:8081/\", host: \"127.0.0.1\"","container_id":"04360d4414bc059034ee511576a1604651aa2df4ab6bcf872ea1de5e8be443ff","container_name":"/unms-nginx","source":"stderr"}

output of docker ps:

`CONTAINER ID        IMAGE                    COMMAND                  CREATED             STATUS              PORTS                                      NAMES
04360d4414bc        ubnt/unms-nginx:0.11.3   "/entrypoint.sh ngin…"   8 minutes ago       Up 8 minutes        0.0.0.0:80->80/tcp, 0.0.0.0:443->443/tcp   unms-nginx
777e3d102470        ubnt/unms:0.11.3         "/usr/bin/dumb-init …"   8 minutes ago       Up 8 minutes                                                   unms
adab502b207b        postgres:9.6.1-alpine    "/docker-entrypoint.…"   8 minutes ago       Up 8 minutes                                                   unms-postgres
3305672adbb9        redis:3.2.8-alpine       "docker-entrypoint.s…"   8 minutes ago       Up 8 minutes                                                   unms-redis
54749209b614        rabbitmq:3               "docker-entrypoint.s…"   8 minutes ago       Up 8 minutes                                                   unms-rabbitmq
1cd9cc4aad03        unms_fluentd             "/entrypoint.sh /bin…"   8 minutes ago       Up 8 minutes        5140/tcp, 127.0.0.1:24224->24224/tcp       unms-fluentd`

output of:
`docker inspect unms --format="{{.Config.Cmd}}"`

`[npm start]
`

**Comment by jflidr** [comment]
@fgrosswig It's not a problem of NGINX but UNMS container didn't respond. Could you please check [UNMS logs](https://help.ubnt.com/hc/en-us/articles/115015690107-UNMS-Logs-and-Bug-Reporting) or send them to me jindrich.flidr(at)ubnt.com.

**Comment by jflidr** [comment]
@fgrosswig Could you please try to login again? It looks like that you tried to login to UNMS when server was starting. It takes a minute. I suppose that you didn't reload page with UNMS in your browser and you saw a login screen instead of server starting page.

**Comment by Ve2mrx** [comment]
I'm having a similar problem. It won't respond to it's normal URL, even if I try 20 minutes after starting the VM.
`
Description:    Ubuntu 16.04.3 LTS
Release:        16.04
Codename:       xenial
`
Running inside VirtualBox on Windows 7 x64 

All up to date as of 2018-02-13

After restart, `docker run hello-world` runs, but ` sudo docker ps
CONTAINER ID        IMAGE               COMMAND             CREATED             STATUS              PORTS               NAMES
`
If I stop and start or `sudo ~unms/app/unms-cli restart`, it works.

I took two support packages, une before a bunch of tests and another after. I will email them to jindrich.flidr(at)ubnt.com with a link to this post.

I wonder if a recent update to the docker-ce package might have a link?

**Comment by radekskrivan** [comment]
@Ve2mrx  Unfortunately we didn't find anything wrong in the logs. We tried to simulate the issue but everything seems to be working fine. 
Can you describe your system in more details, please? You did a fresh clean install just with UNMS or are there some other applications?

**Comment by marcusmarcal** [comment]
Same problem here... don't start after reboot :-(

**Comment by Ve2mrx** [comment]
[apt-installed.txt](https://github.com/Ubiquiti-App/UNMS/files/1725020/apt-installed.txt)


On a hunch, I downgraded the Docker-ce package to the latest **STABLE** release, `17.12.0~ce-0~ubuntu` and rebooted. Works. UNMS starts up properly after reboot.

It seems the latest **EDGE** package `18.02.0~ce-0~ubuntu` causes issues with UNMS startup. 

Since the EDGE channel is the default in [https://get.docker.com/](https://get.docker.com/), when called from install-full.sh it is set to EDGE in the OS. 

In install-full.sh:
```
install_docker() {
  if ! which docker > /dev/null 2>&1; then
    echo "Download and install Docker"
    curl -fsSL https://get.docker.com/ | sh
  fi
```

In get.docker.com:

```
# This value will automatically get changed for:
#   * edge
#   * test
#   * experimental
DEFAULT_CHANNEL_VALUE="edge"
if [ -z "$CHANNEL" ]; then
	CHANNEL=$DEFAULT_CHANNEL_VALUE
fi
```

Maybe asking which `$CHANNEL` to use could help?

**Comment by radekskrivan** [comment]
Thank you for detailed info, @Ve2mrx. We will fix this issue in 0.12.0.

**Comment by elviking** [comment]
Had the same problem for the last couple of versions. @Ve2mrx's fix worked for me.

**Comment by Ve2mrx** [comment]
Update: Docker 18.03.0-ce (2018-03-21) has been pushed to Stable channel. Downgrading is the norm now?

When will 0.12.0 be released?

**Comment by telx85** [comment]
It is now mate: https://community.ubnt.com/t5/UNMS-Beta/UNMS-version-0-12-0-has-been-released/td-p/2295482

**Comment by Ve2mrx** [comment]
I can confirm 0.12.0 is working for me, on `docker-ce_18.03.0~ce-0~ubuntu`!
